.github/build.sh

```diff
@@ -125,17 +125,12 @@ case $compileMode in
     cd ..
     if [ "$stMode" = true ]; then
       # Set CMake flags for ST mode
-      # Fix for the Cmake executing build of the sdk which errors out linking incorrectly to openssl
```

Comment:
Have we tested if these changes were compatible to the secure tunneling build? it seem like we introduced it to fix some openssl issue. https://github.com/awslabs/aws-iot-device-client/commit/da7033372c243720d6fddec691373cbf45dd5903